src/Main.cpp

```diff
@@ -15,18 +15,24 @@
 // You should have received a copy of the GNU General Public License
 // along with this program; if not, write to the Free Software
 // Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#include <string>
+#include <fstream>
+#include <streambuf>
 
 #include "PluginDefinition.h"
 #include "Version.h"
 #include "AboutDialog.h"
+#include "LanguageDialog.h"
 #include "resource.h"
 #include "Config.h"
 #include "ScintillaGateway.h"
 #include "Utilities.h"
+#include "menuCmdID.h"
 
 static HANDLE _hModule;
 static NppData nppData;
 static Configuration config;
+static std::map<std::string, std::string> bufferLanguages;
```

Comment:
It is probably easier to make the keys `std::wstring` that way it saves converting to/from UTF8.